maestro-app/src/main/resources/analysis_centric.json

```diff
@@ -0,0 +1,198 @@
+            "search_analyzer": "lowercase_keyword",
+            "type": "text"
+          }
+        },
+        "type": "keyword"
+      },
+      "analysis_id": {
+        "type": "keyword",
+        "copy_to": [
+          "file_autocomplete"
+        ]
+      },
+      "analysis_type": {
+        "properties": {
+          "name": {
+            "type": "keyword"
+          },
+          "version": {
+            "type": "keyword"
+          }
+        }
+      },
+      "analysis_state": {
+        "type": "keyword"
+      },
+      "study_id": {
+        "type": "keyword",
+        "copy_to": [
+          "file_autocomplete"
+        ]
+      },
+      "donors": {
+        "type": "nested",
+        "properties": {
+          "id": {
+            "type": "keyword",
+            "copy_to": [
+              "file_autocomplete"
+            ]
+          },
+          "submitted_id": {
+            "type": "keyword",
+            "copy_to": [
+              "file_autocomplete"
+            ]
+          },
+          "specimen": {
+            "type": "nested",
+            "properties": {
+              "id": {
+                "type": "keyword",
+                "copy_to": [
+                  "file_autocomplete"
+                ]
+              },
+              "submitted_id": {
+                "type": "keyword",
+                "copy_to": [
+                  "file_autocomplete"
+                ]
+              },
+              "type": {
+                "type": "keyword",
+                "copy_to": [
+                  "file_autocomplete"
+                ]
+              },
+              "samples": {
+                "type": "nested",
+                "properties": {
+                  "id": {
+                    "type": "keyword",
+                    "copy_to": [
+                      "file_autocomplete"
+                    ]
+                  },
+                  "submitted_id": {
+                    "type": "keyword",
+                    "copy_to": [
+                      "file_autocomplete"
+                    ]
+                  },
+                  "type": {
+                    "type": "keyword",
+                    "copy_to": [
+                      "file_autocomplete"
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "files": {
+        "properties": {
+          "id": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          },
+          "size": {
+            "type": "keyword"
+          },
+          "type": {
+            "type": "keyword"
+          },
+          "md5sum": {
+            "type": "keyword"
+          },
+          "access": {
+            "type": "keyword"
+          },
+          "data_type": {
+            "type": "keyword"
+          }
+        }
+      },
+      "experiment": {
+        "type": "object"
+      },
+      "repositories": {
```

Comment:
this is missing url

Reply:
`url` will be added as part of ticket#103